lib/models/state.js

```diff
@@ -1138,6 +1138,27 @@ class State extends new Record(DEFAULTS) {
   }
 
   /**
+   * Wrap the current selection with prefix/suffix.
+   * Cursor will be moved before the suffix
```

Comment:
I think we should make the cursor not change places, similar to how the other wrapping ones maintain the selection as it appears to the user. So if it starts as:

```
|text|
```

And is wrapped with `**` it would become:

```
**|text|**
```

Does that make sense?